src/components/KTable/KTable.vue

```diff
@@ -632,6 +680,114 @@ const tdlisteners = computed((): any => {
   }
 })
 
+const columnWidths = ref<Record<string, number>>({})
```

Comment:
question: is there a reason you initialize with an empty object instead of just undefined `ref()` ?

Reply:
It makes it easier to not have to check if the object is defined or not down below when I start adding entries for the column widths.